source/views/home/index.tsx

```diff
@@ -21,54 +28,121 @@ const styles = StyleSheet.create({
+		const OnPressMenuItem = (result: string) => {
+			const typedResult = result as keyof typeof HomeNavbarButtonsEnum
+			switch (typedResult) {
+				case HomeNavbarButtonsEnum.Settings:
+					navigation.navigate('Settings')
+					break
+				case HomeNavbarButtonsEnum.ChangeBackground: {
+					appBackground.mutate()
+					break
+				}
+				default:
+					console.warn(`Unhandled tap on settings button item: ${result}`)
+					break
+			}
+		}
+
+		return (
+			<ContextMenu
+				actions={Object.values(HomeNavbarButtonsEnum)}
+				isMenuPrimaryAction={true}
+				onPressMenuItem={OnPressMenuItem}
+				title=""
+			>
+				<OpenSettingsButton canGoBack={true} />
+			</ContextMenu>
+		)
+	}, [appBackground, navigation])
+
+	React.useLayoutEffect(() => {
+		navigation.setOptions({
+			headerRight: () => SettingsContextButton,
+		})
+	}, [SettingsContextButton, navigation])
+
 	return (
-		<ScrollView
-			alwaysBounceHorizontal={false}
-			showsHorizontalScrollIndicator={false}
-			showsVerticalScrollIndicator={false}
-			testID="screen-homescreen"
-		>
-			<View style={styles.cells}>
-				{columns.map((contents, i) => (
-					<Column key={i} style={styles.column}>
-						{contents.map((view) => (
-							<HomeScreenButton
-								key={view.type === 'view' ? view.view : view.title}
-								onPress={() => {
-									if (view.type === 'url') {
-										return openUrl(view.url)
-									} else if (view.type === 'view') {
-										return navigation.navigate(view.view)
-									} else {
-										throw new Error(`unexpected view type ${view.type}`)
-									}
-								}}
-								view={view}
-							/>
-						))}
-					</Column>
-				))}
-			</View>
+		<SafeAreaView style={styles.safearea}>
+			<ScrollView
+				alwaysBounceHorizontal={false}
+				contentInsetAdjustmentBehavior="automatic"
+				showsHorizontalScrollIndicator={false}
+				showsVerticalScrollIndicator={false}
+				testID="screen-homescreen"
+			>
+				<View style={styles.cells}>
+					{columns.map((contents, i) => (
+						<Column key={i} style={styles.column}>
+							{contents.map((view) => (
+								<HomeScreenButton
+									key={view.type === 'view' ? view.view : view.title}
+									onPress={() => {
+										if (view.type === 'url') {
+											return openUrl(view.url)
+										} else if (view.type === 'view') {
+											return navigation.navigate(view.view)
+										} else {
+											throw new Error(`unexpected view type ${view.type}`)
+										}
+									}}
+									view={view}
+								/>
+							))}
+						</Column>
+					))}
+				</View>
 
-			<UnofficialAppNotice />
-		</ScrollView>
+				<UnofficialAppNotice />
+			</ScrollView>
+			{Boolean(appBackgroundImage) && (
+				<ImageBackground
+					resizeMode="cover"
+					source={{uri: appBackgroundImage}}
+					style={styles.background}
+				/>
+			)}
+		</SafeAreaView>
 	)
 }
 
+const HomeNavbarButtonsEnum = {
+	Settings: 'Settings',
+	ChangeBackground: 'Change background',
```

Comment:
```suggestion
	ChangeBackground: 'Change Background',
```

I feel like Title Case is most often used in iOS context menus?